Source/ConversationV1/Models/MessageRequest.swift

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright IBM Corporation 2016
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import Foundation
+import Freddy
+
+//Struct for Messages
+public struct MessageRequest: JSONEncodable, JSONDecodable {
+    
+    var input:   [String: JSON]!
```

Comment:
In general, we want to avoid using `JSON` as the type of any model properties. I know Conversation is a little odd... but I think we can probably avoid it here.

Instead of declaring the type as `[String: JSON]` what do you think about `InputData` with the following definition:

```swift
internal struct InputData: JSONEncodable {
    
    private let text: String

    internal init(text: String) {
        self.text = text
    }

    internal func toJSON() -> JSON {
        return .Dictionary(["text": text])
    }
}
```

This more-closely matches the documentation on the Swagger page and would make it easier to update in the future.